### DISCUSSION TITLE
Vercel Account Creation

### DISCUSSION THREAD

**Discussion by ref-err** [vercel/community#3156] [Help] [answered]
### Summary

Hello! So, when i creating my vercel account, its forcing me to type my phone number, but when i typing it, it says that number's blocked. I was registering for the first time, what's problem? I didn't do anything wrong.

### Steps to Reproduce

Unblock my phone number.

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the  email, phone number, and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
